NitroxServer/Communication/Packets/Processors/VehiclePositionFixProcessor.cs

```diff
@@ -0,0 +1,23 @@
+﻿using NitroxModel.Logger;
+using NitroxModel.Packets;
+using NitroxServer.Communication.Packets.Processors.Abstract;
+using NitroxServer.GameLogic.Vehicles;
+
+namespace NitroxServer.Communication.Packets.Processors
+{
+    class VehiclePositionFixProcessor : AuthenticatedPacketProcessor<VehiclePositionFix>
+    {
+        private readonly VehicleManager vehicleManager;
+
+        public VehiclePositionFixProcessor(VehicleManager vehicleManager)
+        {
+            this.vehicleManager = vehicleManager;
+        }
+
+        public override void Process(VehiclePositionFix packet, Player player)
+        {
+            Log.Info(packet);
+            vehicleManager.UpdateVehiclePosition(packet.VehicleId, packet.Position);
```

Comment:
What is the purpose of this?  I think we are already broadcasting vehicle locations?

Reply:
The server only stores the vehicle's spawn position and if no one is using the vehicle no VehicleMovementPacket will be send and the server will never have informations about the latest position once the vehicle stopped falling.
In the case of cyclops/seamoth it's not much a problem since the vehicle will just fall from the sky, but it is more problematic with the exosuits which can get stuck in the air and much too problematic for rocketBase which are no longer subject to gravity when you start building the rocket

Reply:
wouldnt it be more beneficial code wise to change the way the vehicleMovementPacket is handled? instead of creating a different class that might or might not cause some positional issues when multiple packets handling the same type of data are recieved?